scripts/make_sfc_climo.py

```diff
@@ -0,0 +1,112 @@
+"""
+The run script for sfc_climo_gen 
+"""
+
+import glob
+import os
+import sys
+from argparse import ArgumentParser
+from pathlib import Path
+
+from uwtools.api.fs import link as uwlink
+from uwtools.api.sfc_climo_gen import SfcClimoGen 
+from uwtools.api.config import get_yaml_config
+
+
+def link_files(dest_dir, files):
+    """
+    Link a given list of files to the destination directory updating the file names.
+    """
+    for fpath in files:
+        path = Path(fpath)
+        fn = Path(fpath).name
+    
+        if "halo" in fn:
+            fn = f"{CRES}.{(fn.replace('halo', 'halo4'))}"
+            no_halo_fn = fn.replace("halo4.", "")
+            for link in (fn, no_halo_fn):
+                link = Path(link)
+                if (linkname := dest_dir / link.name).is_symlink():
+                    linkname.unlink()
+                linkname.symlink_to(path)
+    
+        else:
+            bn = fn.split(".nc")[0]
+            fn = f"{CRES}.{bn}.halo0.nc"
+            tile1_fn = fn.replace("tile7.halo0", "tile1")
+            for link in (fn, tile1_fn):
+                link = Path(link)
+                if (linkname := dest_dir / link.name).is_symlink():
+                    linkname.unlink()
+                linkname.symlink_to(path)
+
+
+parser = ArgumentParser(
+    description="Script that runs sfc_climo_gen via uwtools API.",
+)
+parser.add_argument(
+    "-c",
+    "--config-file",
+    metavar="PATH",
+    required=True,
+    help="Path to experiment config file.",
+    type=Path,
+)
+parser.add_argument(
+    "--key-path",
+    help="Dot-separated path of keys leading through the config to the driver's YAML block.",
+    metavar="KEY[.KEY...]",
+    required=True,
+    type=lambda s: s.split("."),
+)
+
+args = parser.parse_args()
+expt_config = get_yaml_config(args.config_file)
+
+# The experiment config will have {{ CRES | env }} expressions in it that need to be
+# dereferenced during driver initialization
+CRES = expt_config["workflow"]["CRES"]
+os.environ["CRES"] = CRES 
+expt_config.dereference(
+    context={
+        **os.environ,
+        **expt_config,
+    }
+)
```

Comment:
Working on wrapping this into a function. I was able to pass the fundamental test suites without a `parse_args `function. I haven't had any luck in wrapping it into a function and passing the fundamental test suite. This part is still in the works to match the `make_orog` [PR](https://github.com/christinaholtNOAA/ufs-srweather-app/blob/911842160bca58653131a099c2b21ffd72442ff1/scripts/make_orog.py) that merged last week.